src/components/Heroes/rankColumns.ts

```diff
@@ -12,7 +12,7 @@ type Props = {
 
 enum HeroesTab {
   PRO = 'pro',
-  PUBLIC = 'public'
+  PUBLIC = 'public',
```

Comment:
we need TURBO here right?

Reply:
related to comment below: if we'll use object sintax, then TURBO here is needed.